gtsfm/data_association/point3d_initializer.py

```diff
@@ -210,9 +210,9 @@ def triangulate(self, track_2d: SfmTrack2d) -> Tuple[Optional[SfmTrack], Optiona
             self.track_camera_dict, triangulated_pt, inlier_track.measurements
         )
 
-        # Check that all the measurements have reprojection error < threshold.
-        # TODO(johnwlambert): compare with approach where we only throw away the outlier measurements.
-        if not np.all(reproj_errors < self.reproj_error_thresh):
+        # Filter measurements by the reprojection error.
```

Comment:
maybe these lines below are more readable on 2 lines? like 
```python
valid_reproj_error_idxs = np.where(reproj_errors.flatten() < self.reproj_error_thresh)[0]
inlier_track = inlier_track.select_subset(valid_reproj_error_idxs)
```
maybe there's a better name for this. like instead of `valid_reproj_error_idxs`, something like `idxs_reproj_err_satisfied`?

Reply:
Done